drivers/ethernet/eth_stm32_hal.c

```diff
@@ -72,6 +167,15 @@ static inline void disable_mcast_filter(ETH_HandleTypeDef *heth)
 {
 	__ASSERT_NO_MSG(heth != NULL);
 
+#if defined(CONFIG_SOC_SERIES_STM32H7X)
+	ETH_MACFilterConfigTypeDef MACFilterConf;
```

Comment:
This addition looks very crowded, some more empty lines would do good.

Reply:
> This addition looks very crowded, some more empty lines would do good.

done

Reply:
added empty lines